go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -443,7 +444,46 @@ func handleNetdevResources(resourceName string) (string, error) {
 	return netdevice, nil
 }
 
-func createNodeManagementPorts(name string, nodeAnnotator kube.Annotator, waiter *startupWaiter,
+func exportManagementPortAnnotation(link netlink.Link, nodeAnnotator kube.Annotator) error {
```

Comment:
Code is very similar to addDPUConnectionDetailsAnnot in cni_dpu.go. We might want to create a common function.

Reply:
Created a new function to get pf from vf address.